src/components/section/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC, ReactNode } from 'react';
+
+import { Box, Heading, Text } from '@chakra-ui/react';
+
+import Stack from '../stack';
+
+interface Props {
+  variant?: 'hero';
+  title: string;
+  text?: string;
+  image?: () => ReactNode;
+}
+
+const Section: FC<Props> = ({ title, text, image, variant = 'hero' }) => {
```

Comment:
[Observation] we likely don't want `hero` to be a default variant since we would use `default` one more often. Switching the default value would be a breaking change. How about we avoid this and make `variant` mandatory for now?